src/components/paintingCard/PaintingCard.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
```

Comment:
you don't need to import the `Component` for `functional components`

Reply:
OMG! I didn't take that out...I initially started this component as class-based.  I will fix that, thank you!